internal/interpolate/generated.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+echo -e "// Code generated by ragel\n// @generated" | cat - parse.go > tmp
```

Comment:
I'm not sure I understand the reasoning for that. The "generated by X" convention usually only references the tool that was used to generate the code, not the library for which it was generated. The exception to this was the ThriftRW plugin for YARPC because it did a lot of extra work in addition to ThriftRW.